pkg/controller/operators/catalog/operator.go

```diff
@@ -1423,6 +1445,23 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 	return
 }
 
+func (o *Operator) requeueSubscriptionForInstallPlan(plan *v1alpha1.InstallPlan, logger *logrus.Entry) {
+	// Notify subscription loop of installplan changes
+	owners := ownerutil.GetOwnersByKind(plan, v1alpha1.SubscriptionKind)
+
+	if len(owners) == 0 {
+		logger.Trace("no installplan owner subscriptions found to requeue")
+		return
+	}
+
+	for _, owner := range owners {
```

Comment:
can an installplan ever be owned by more than 1 subscription -- maybe we want to explicitly check `len(owners)`?

Reply:
It can (https://github.com/operator-framework/operator-lifecycle-manager/blob/e5a160aa2a69b4afe3a8da0add98b4083975096c/pkg/controller/operators/catalog/operator.go#L1143), since all Subscriptions in a namespace are resolved together.

Reply:
This is coming over from an existing block in the sync that I didn't really modify.

But even if we have >1 owners for an InstallPlan, it's okay to requeue them all right? Since they need to update their status to reflect the change in the InstallPlan phase.

So @exdx not sure I understood what you meant by explicitly checking `len(owners)`.

Reply:
That sounds fine. I guess I am conceptually confused by the case where an installplan would have multiple subscriptions associated with it -- does it introduce any weird edge cases? But it looks like our code has no issues with that. 

As @hasbro17 it's fine to requeue them all since they would all need to be updated with the new installplan status